app/controllers/close_contacts_controller.rb

```diff
@@ -6,7 +6,8 @@ class CloseContactsController < ApplicationController
 
   # Create a new close contact
   def create
-    redirect_to root_url && return unless current_user.can_create_patient_close_contacts?
+    redirect_to(root_url) && return unless current_user.can_create_patient_close_contacts?
```

Comment:
These are good changes but while we're here why not also just make these use `and` rather than `&&` like they're technically supposed to use?

Reply:
`and` is a stylistic choice vs. `&&` and based on our Rubocop configs the codebase has chosen to use `&&`

Reply:
The docs don't seem to consider it stylistic (from the link in your description):
> Make sure to use and return instead of && return because && return will not work due to the operator precedence in the Ruby Language.


given `and` functions differently than `&&`. I don't fully understand why `&&` is even working here to be honest, but it seems like best practice is to use `and`. I've done this elsewhere since I learned the above and Rubocop does not complain.

Reply:
the rails docs aren't being prescriptive. It's just informing of a gotcha if one were to substitute `and` for `&&` based on stylistic choice of the reader

Reply:
Yeah, I guess I was misunderstanding the assertion made in the docs (and reading done elsewhere) - I thought it was saying that even with the parens there could be issues with using && in these cases but I suppose that’s just not true and the order of evaluation only becomes an issue when there’s no parens, which makes more sense. Nevermind!